generators/client/templates/react/src/main/webapp/app/modules/administration/configuration/configuration.tsx

```diff
@@ -40,77 +49,104 @@ export class ConfigurationPage extends React.Component<IConfigurationPageProps,
 
   filterFn = prefix => prefix.toUpperCase().includes(this.state.filter.toUpperCase());
 
+  sortPrefixFn = (prop1, prop2) => {
+    const { configuration } = this.props;
+    if (configuration && configuration.configProps) {
+      const pref1 = this.props.configuration.configProps[prop1].prefix;
```

Comment:
use `configuration` const here and next line instead of `this.props.configuration`